web_tree_resize_column/static/src/js/backend.js

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2019 Onestein
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_tree_resize_column.backend', function (require) {
+    "use strict";
+
+    var ListRenderer = require('web.ListRenderer');
+
+    ListRenderer.include({
+
+        /**
+         * @override
+         */
+        _renderView: function() {
+            // Preserve width of columns
+            var styles = [];
+            this.$el.find('thead th').each(function () {
+                styles.push($(this).attr('style'));
+            });
+
+            var res = this._super.apply(this, arguments);
+
+            // Initialize jQuery plugin
+            this.$el.find('table').resizableColumns();
+
+            // Restore width of columns
+            this.$el.find('thead th').each(function (index, th) {
```

Comment:
Can't reuse previous "find"?

Reply:
No, because after the super the element does not exists anymore.